include/ametsuchi/pager/page.h

```diff
@@ -17,14 +17,18 @@
 
 #pragma once
 
-#include <stdint.h>  // for uintX_t
+#include <ametsuchi/globals.h>
+#include <math.h>
 
 namespace ametsuchi {
 namespace pager {
 
+#pragma pack(push, 1)
 
-enum class PageType : uint8_t { DATA_ITEM, MERKLE_NODE, INDEX_NODE };
+enum { AMETSUCHI_PAGE_SIZE = 4096, };
```

Comment:
That specific to pager, only a small part of low-level components (instances of VFS) actually need this
globals.h -> file with definition of API-level